polaris/ocean/model/ocean_model_step.py

```diff
@@ -112,9 +113,14 @@ def setup(self):
             self.config_models = ['ocean', 'Omega']
             self.yaml = 'omega.yml'
             self._read_map()
+            self.partition_graph = False
         elif model == 'mpas-ocean':
             self.config_models = ['ocean', 'mpas-ocean']
             self.make_yaml = False
+            self.add_input_file(
+                filename='graph.info',
+                work_dir_target=self.graph_target)
```

Comment:
With my suggestion above, this is just:
```suggestion
                work_dir_target=graph_target)
```